package.json

```diff
@@ -50,7 +50,8 @@
   "dependencies": {
     "fs": "0.0.1-security",
     "mongoose": "^4.7.8",
-    "path": "^0.12.7"
+    "path": "^0.12.7",
+    "socket.io": "^1.7.2"
```

Comment:
We will have to discuss this because I'm not sure but I think we are going to remove ``socket.io`` and force the client to provide a socket.